docker/openproblems-python37-scgen/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM condaforge/mambaforge
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+RUN \
+    apt-get update && \
+    apt-get install -qy sudo git unzip&& \
+    useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
+    chmod g+w /etc/passwd && \
+    echo "${NB_USER}    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers && \
+    # Prevent apt-get cache from being persisted to this layer.
+    find /var/lib/apt/lists/ ! -type d -exec rm '{}' \;
+
+RUN mamba install -y python=3.7 && mamba install -c conda-forge r-biocmanager && mamba install -c bioconda -y bioconductor-scran
+
+# update pip
+RUN python3 -m pip install --no-cache-dir -U pip
+
+# install AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip -q awscliv2.zip
+RUN ./aws/install
+
+# install dependencies
+COPY ./docker/openproblems-python37-scgen/requirements.txt ./requirements.txt
+RUN pip install --no-cache-dir "numpy==1.18.5"
+RUN pip install --no-cache-dir git+https://github.com/theislab/scib
+RUN pip install --no-cache-dir scgen
+RUN pip install --no-cache-dir desc
```

Comment:
desc and scgen in the same container? does this work? I thought they have clashing tensorflow dependencies?